content/usecaseItems.md

```diff
@@ -9,23 +11,24 @@ example_usecase:
     route: /usecase_0/panel_1
     image: /usecase_0/img_1.png
     
-  # - title: Data used in this study
-  #   id: 2
-  #   route: /usecase_0/panel_2
-  #   image: /usecase_0/img_2.png
+  - title: Data used in this study
```

Comment:
I still don't like the repetition from the use cases themselves. Since all we really need is the figures, can't we just do something like `for img in '/usecase_0/*'`?

Reply:
Now the images are loaded from each .md file.